README.md

```diff
@@ -32,11 +32,13 @@ steps:
       approvers: user1,user2,org-team1
       minimum-approvals: 1
       issue-title: "Deploying v1.3.5 to prod from staging"
+      exclude-workflow-initiator-as-approver: false # Optional, defaults to false
```

Comment:
I think this is the only pending change prior to a merge. Instead of putting the comment "Optional, defaults to false" here, could you move that to line 41 and add this default information? Thanks!

Reply:
Done!